python/sidecar/http.py

```diff
@@ -0,0 +1,50 @@
+import sidecar
+from fastapi import FastAPI
+from sidecar._typing import (RewriteMannerType, RewriteRequest,
+                             RewriteResponse, TextCompletionRequest,
+                             TextCompletionResponse)
+
+app = FastAPI()
+
+
+@app.get("/")
+async def index():
+    return {"message": "Hello World"}
+
+
+@app.get("/rewrite")
```

Comment:
We'll want all these routes to live under `/api` or even `/api/v0` to avoid collisions with static content handlers.